.github/workflows/vulnerability-scanning-on-repo.yml

```diff
@@ -59,6 +63,14 @@ jobs:
         with:
           name: ${{ github.run_number }}${{ inputs.package-directory }}.sbom.json
           path: ${{ inputs.package-directory }}/${{ env.BOM_FILE_NAME }}
+      - name: Upload trivyignore
+        if: ${{ inputs.TRIVY_IGNORES }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ github.run_number }}.trivyignore
+          if-no-files-found: error
+          include-hidden-files: true
+          path: ${{ inputs.package-directory }}/${{ inputs.TRIVY_IGNORES }}
```

Comment:
Same comment as above, Ignore files **SHOULDN'T** be enabled or encouraged in our workflows